src/webhook/mutation/pod_mutator/oneagent_mutation/annotations.go

```diff
@@ -3,17 +3,18 @@ package oneagent_mutation
 import (
 	"net/url"
 
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
 	"github.com/Dynatrace/dynatrace-operator/src/kubeobjects"
 	dtwebhook "github.com/Dynatrace/dynatrace-operator/src/webhook"
 	corev1 "k8s.io/api/core/v1"
 )
 
 type installerInfo struct {
-	flavor        string
-	technologies  string
-	installPath   string
-	installerURL  string
-	failurePolicy string
```

Comment:
what happened to the `failurePolicy`?

Reply:
its redundant here, its not a oneagent specific thing so it still present but not in the `oneagent_mutation`

Reply:
here: https://github.com/Dynatrace/dynatrace-operator/blob/master/src/webhook/mutation/pod_mutator/init_container.go#L23